Content.Client/Security/Ui/GenpopLockerMenu.xaml.cs

```diff
@@ -16,14 +16,11 @@ public sealed partial class GenpopLockerMenu : FancyWindow
 
     public event Action<string, float, string>? OnConfigurationComplete;
 
-    // CCVar.
-    private int _maxIdJobLength;
-
     public GenpopLockerMenu(EntityUid owner, IEntityManager entMan)
     {
         RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
```

Comment:
This is part of why I think all controls should have blank ctors so we can just have a test sweep all controls, and construct them just to check their dependencies actually loaded.